kumascript/src/info.js

```diff
@@ -39,15 +39,18 @@ const info = {
     return new URL(url, DUMMY_BASE_URL).pathname.replace(/\/$/, "");
   },
 
-  cleanURL(url) {
+  cleanURL(url, followRedirects = true) {
     // This function returns just the lowercase pathname of the given "url",
     // removing any trailing "/". The DUMMY_BASE_URL is not important here, since
     // we're only after the path of any incoming "url", but it's required by
     // the URL constructor when the incoming "url" is relative.
-    const uri = new URL(url, DUMMY_BASE_URL).pathname
-      .replace(/\/$/, "")
-      .toLowerCase();
-    return Redirect.resolve(repairURL(uri));
+    const repairedURL = repairURL(
+      new URL(url, DUMMY_BASE_URL).pathname.replace(/\/$/, "").toLowerCase()
+    );
+    if (followRedirects) {
+      return Redirect.resolve(repairedURL);
```

Comment:
As you did in the PR that this replaces, should we call `repairURL()` on the the redirect that this potentially returns? So something like you did before:
```js
      resultURL = Redirect.resolve(repairedURL);
      if (resultURL !== repairedURL) {
        return repairURL(resultURL);
      }
      return resultURL;
```
I'm thinking we probably should, because the `_redirects.txt` will possibly be edited by humans.

Reply:
I'll carry that over from the, now closed, PR we did yesterday.

Reply:
Done. r?